services/ui-src/src/components/export/ExportedReportFieldTable.tsx

```diff
@@ -45,24 +46,45 @@ export const ExportedReportFieldTable = ({ section }: Props) => {
   const reportType = report?.reportType as ReportType;
   const hideHintText = reportType === ReportType.MLR;
 
-  // handle ILOS rendering logic
-  const renderIlosVerbiage = () => {
-    const hasIlos = report?.fieldData["ilos"]?.length;
-    return section.path === "/mcpar/plan-level-indicators/ilos" && !hasIlos;
-  };
-
   const hasPlans = report?.fieldData["plans"]?.length;
+  const hasIlos = report?.fieldData["ilos"]?.length;
+  const hasBss = report?.fieldData["bssEntities"]?.length;
+
+  // handle missing plans / ilos rendering logic
+  const renderMissingEntityVerbiage = () => {
+    let missingVerbiage;
```

Comment:
Find this method a little hard to read. Suggested refactor:
```
const { verbiage } = (section as DrawerReportPageShape);

// verbiage for ILOS
if (section.path === "/mcpar/plan-level-indicators/ilos" && !hasIlos) {
  if (!hasPlans) {
    return verbiage.missingPlansAndIlosMessage;
  }
  return verbiage.missingIlosMessage;
}

// verbiage for missing plans
if (!hasPlans) {
  return verbiage.missingEntityMessage;
}
return;
```

Reply:
That's super helpful, I just committed that refactor. Thank you!!